index.html

```diff
@@ -0,0 +1,167 @@
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+        
+  </div>
+ 
+    <div class="front-heading">
+        <div class="front-content">
+                <h1 class="hello">"Hello Sharing world"</h1>
+                <h1 class="big-heading">CREATIVE COMMONS <br>GLOBAL SUMMIT <br>2015</h1>
+
+            <div class="heading-par">
+                <p>
+                    A joyful celebration believing in the value of openness and <br>sharing, creating 
+                    a positive change with people from over 80 <br>countries is taking place in October, in Korea.
+                </p>
+            </div>
+            <h1 class="date-txt">2015.10.1 (THU) ~ 16(FRI)</h1>
+            <h1 class="place-txt">@ National Museum of Korea, Art Center Nabi and more </h1>
+
+    
+       </div>
+    </div>
+</header> 
+<section class="main-program">
+    <h2 class="container-heading">Main Program</h2>
+    <span class="red-bar"></span>
+    <div class="card-container">
+            <div class="card">
+            <div class="card-icon">
+                <img class="card-icon" src="./images/icons/icon1.png" />
+            </div>
+                <h3 class="program-name">Lecture</h3>
+                <p class="program-detail">
+                  Listen to the speakers from
+                  various countries about
+                  the messgae of sand opening
+                </p>
+            </div>               
+            
+            <div class="card">
+                <div class="card-icon">
+                    <img class="card-icon" src="./images/icons/icon2.png" />
+                </div>
+                    <h3 class="program-name">CC Exhibition</h3>
+                    <p class="program-detail">
+                      Listen to the speakers from
+                      various countries about
+                      the messgae of sand opening
+                    </p>
+                </div>     
+                <div class="card">
+                    <div class="card-icon">
+                        <img class="card-icon" src="./images/icons/icon3.png" />
+                    </div>
+                    <h3 class="program-name">Forum</h3>
+                    <p class="program-detail">
+                    Listen to the speakers from
+                    various countries about
+                    the messgae of sand opening
+                    </p>
+                    </div>     
+                    <div class="card">
+                <div class="card-icon">
+                    <img class="card-icon" src="./images/icons/icon4.png" />
+                </div>
+                    <h3 class="program-name">Workshop</h3>
+                    <p class="program-detail">
+                        Listen to the speakers from
+                        various countries about
+                        the messgae of sand opening
+                    </p>
```

Comment:
- [ OPTIONAL] Kindly remove the black background behind your program list as seen below, in order to stick to Cindy's design:
![Screenshot from 2023-02-17 23-47-39](https://user-images.githubusercontent.com/97834160/219800108-5c464971-6e04-41fd-af95-3b53f5d2eca5.png)